executor/aggregate.go

```diff
@@ -233,6 +243,15 @@ func (e *HashAggExec) Close() error {
 		if e.memTracker != nil {
 			e.memTracker.ReplaceBytesUsed(0)
 		}
+		if e.listInDisk != nil {
+			if err := e.listInDisk.Close(); err != nil {
+				return err
```

Comment:
Don't we close the chilrenExec? This may cause leaks.

Reply:
Addressed.